FusionIIIT/applications/online_cms/models.py

```diff
@@ -1,3 +1,129 @@
-# from django.db import models
+from django.db import models
 
-# Create your models here.
+from applications.academic_information.models import Course, Student
+from applications.globals.models import ExtraInfo
+
+
+class CourseDocuments(models.Model):
+    course_id = models.ForeignKey(Course, on_delete=models.CASCADE)
+    upload_time = models.DateTimeField()
+    description = models.CharField(max_length=100)
+    document_name = models.CharField(max_length=40)
+    document_url = models.CharField(max_length=100, null=True)
+
+    def __str__(self):
+        return str(self.course_id, " ", self.document_name)
+
+
+class CourseVideo(models.Model):
+    course_id = models.ForeignKey(Course, on_delete=models.CASCADE)
+    upload_time = models.DateTimeField()
```

Comment:
Make this field auto_now.

Reply:
This is the course id, example Cs308, it should be made primary_key=true not auto

Reply:
@arshadsid `auto_now` is not same as `auto`

Reply:
I am talking aboit time and datetime fields

Reply:
Ok ok, my bad.